articles/provider-quantinuum.md

```diff
@@ -89,6 +89,14 @@ More information can be found in the *System Model H1 Product Data Sheet* found
   - H1-2: `quantinuum.qpu.h1-2`
 - Target Execution Profile: [:::no-loc text="Basic Measurement Feedback":::](xref:microsoft.quantum.target-profiles)
 
+## Quantinuum’s H-Series Emulator in Code with Azure Quantum
+
+Quantinuum’s H-Series Emulator offered in [Code with Azure Quantum] is a statevector based quantum emulator. This emulator uses a realistic physical noise model and generalized error parameters based on the typical performance of a System Model H1 quantum computer. The quantum simulation performed is the same as the System Model H1 Emulator but the classical circuit optimization routine is reduced to increase throughput.
+
+The full System Model H1 Emulator allows user to test and debug circuits design before running on hardware and is available even while the hardware is offline.  
+
+More information on the full System Model H1 Emulator can be found [above](#system-model-h1-emulators) and in the System Model H1 Emulator Product Data Sheet found on the [System Model H1] page.
```

Comment:
Does [System Model H1] need to link somewhere?

Reply:
https://www.quantinuum.com/hardware/h1